src/general-functions/renderNoteCards.js

```diff
@@ -1,5 +1,5 @@
 import NotesCard from "../components/notes-card";
 
-const renderNoteCards = (notes) => notes.map(({title,body,_id,colour,priority,labels}) => <NotesCard key = {_id} title = {title} body = {body} id = {_id} colour = {colour} priority = {priority} labels ={labels} />)
+const renderNoteCards = (notes) => notes.map(({title,body,_id,colour,priority,labels,date}) => <NotesCard key = {_id} title = {title} body = {body} id = {_id} colour = {colour} priority = {priority} labels ={labels} date = {date} />)
```

Comment:
nothing wrong here but consider sending the entire note as prop instead of destructuring its values and then sending it because whenever you have to make api calls you later then you would have to write every property in payload